src/cacheServices/CacheTransactionService.ts

```diff
@@ -42,44 +46,61 @@ export const transferFilter = (transaction: Transaction): boolean => {
 };
 
 /**
- * Parses through list of transactions and casts them to TransferTransactions so we
+ * Parses through list of transactions and casts them to CacheTransferTransaction so we
  * can have access to important transfer details
  * @param {Transaction} transaction
- * @returns {TransferTransaction}
+ * @returns {CacheTransferTransaction}
  */
-export const mapTransfer = (transaction: Transaction): TransferTransaction => {
+export const mapTransfer = (transaction: Transaction): CacheTransferTransaction => {
```

Comment:
This function seems a little long. Can we further extract pieces? Decouple it some more?